### DISCUSSION TITLE
How to add Custom MAIL FROM domain using only code ?

### DISCUSSION THREAD

**Discussion by LAGGOUNE-Walid** [aws/aws-sdk-php#2791] [Q&A] [answered]
Hello , i am allowing my users to use custom domains and custom mail from domain , using this sdk i could only allow them to add custom domain (example.com) i cannot find where i can allow them to add and verifiy the mail from domain "sender@example.com" using code without the aws dashboard.

**Comment by yenfryherrerafeliz** [comment]
Hi @LAGGOUNE-Walid, could you please be more specific about which service you are trying to use through the SDK?.
Here is a list of things that we need to better assist you:
- AWS SDK minor version that you are using.
- A sample code that shows which SDK client you are using and which operation you are doing with said client.
- PHP version running on

I look forward to your answer.

Thanks!

**Reply by LAGGOUNE-Walid** [reply]
Hi , the sdk version is v3 php is 8.1 . i am trying to allow my users to add custom mail from domain using this sdk . the only method i can see it is to go manually to aws dashboard add obtain mx and txt records from custom mail from domain and add them . i want to add custom mail from using aws api (this sdk) .
 this sdk allow me only to add ,obtain and verify  DomainKeys Identified Mail (DKIM) 
  
![Screenshot_2023-10-12_19-09-13](https://github.com/aws/aws-sdk-php/assets/16167585/695e0038-e858-435c-94c0-df3c59769cda)

**Comment by yenfryherrerafeliz** [comment, marked as answer]
@LAGGOUNE-Walid have you tried with createEmailIdentity using the sesv2 client?. [Here](https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-sesv2-2019-09-27.html#createemailidentity) is the documentation regarding this command.

Please let me know if that helps!

Thanks!

[a comment by an automated account is not shown]
